conda-store-server/conda_store_server/server/app.py

```diff
@@ -362,6 +362,12 @@ def start(self):
             process.start()
 
         try:
+            # Note: the logger needs to be defined here for the output to show
+            # up, self.log doesn't work here either
+            logger = logging.getLogger("app")
+            logger.setLevel(self.log_level)
+            logger.info(f"Starting server on {self.address}:{self.port}")
```

Comment:
We could move this to `initialize`, then we'll be able to use `self.log`. But it'll be far from where `uvicorn` is actually started.

Reply:
This needs to be updated to use `basicConfig` because this uses the implicit handler.